docs/source/quick-start.mdx

```diff
@@ -0,0 +1,109 @@
+# Quick start
+
+The Hugging Face Hub is the go-to place for sharing machine learning models, spaces, datasets, and metrics. `huggingface_hub` library helps you interact with the Hub without leaving your development environment. You can create and manage repositories easily and download and upload files from the Hub. 
+
+## Install the Hub library
+
+To get started, you should install the `huggingface_hub` library. The library provides functionalities that make interacting with the Hub simple. You can install the library with `pip` or `conda`.
```

Comment:
I would remove this sentence as this is a bit similar to what is mentioned in the first paragraph.
```suggestion
To get started, you should install the `huggingface_hub` library. You can install the library with `pip` or `conda`.
```